src/main/java/ch/njol/skript/lang/SkriptParser.java

```diff
@@ -577,6 +580,39 @@ public boolean hasTag(String tag) {
 				log.printError();
 				return null;
 			}
+			Matcher literalClassInfoMatcher = LITERAL_CLASSINFO_PARSE.matcher(expr);
+			if (literalClassInfoMatcher.matches()) {
+				String literalString = literalClassInfoMatcher.group(1);
+				String classInfoString = literalClassInfoMatcher.group(2);
```

Comment:
what do you think of using names like `unparsedClassInfo` or `classInfoMatch` instead? we can even use named groups in the pattern to refer to them that by name instead of group number.